src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -132,25 +130,55 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             },
             handleError: "{that}.events.onError.fire",
             handlePause: {
-                changePath: "paused",
-                value: true
+                funcName: "fluid.textToSpeech.handlePause",
+                args: ["{that}"]
             },
             handleResume: {
-                changePath: "paused",
-                value: false
+                funcName: "fluid.textToSpeech.handleResume",
+                args: ["{that}"]
             }
         }
     });
 
+    // Issue commands to the speechSynthesis interface after a delay; this
+    // makes the wrapper behave somewhat better when issuing commands, especially
+    // play and pause
+    fluid.textToSpeech.asyncSpeechSynthesisControl = function (control, delay, args) {
+        setTimeout(function () {
+            speechSynthesis[control](args);
+        }, delay);
+    };
+
     fluid.textToSpeech.speak = function (that, speaking) {
         that.events[speaking ? "onStart" : "onStop"].fire();
     };
 
-    fluid.textToSpeech.pause = function (that, paused) {
-        if (paused) {
-            that.events.onPause.fire();
-        } else if (that.model.speaking) {
-            that.events.onResume.fire();
+    fluid.textToSpeech.handlePause = function (that) {
+        that.applier.change("paused", true);
+        that.events.onPause.fire();
```

Comment:
I can't go along with either of these arguments :) The whole purpose of having a model idiom is to avoid the proliferation of "uninterpreted instants in time". You have 2 events and 4 invokers, when you could have a simple flag in a model and a call to a standard algorithm. Ideally we never have "non-model event structure", parallel or otherwise.